src/main/java/com/docker/jersey/restapi/DockerRegistrationApi.java

```diff
@@ -0,0 +1,28 @@
+package com.docker.jersey.restapi;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.Optional;
+
+@Path("/dockerRegistration/")
+public class DockerRegistrationApi {
+    @POST
+    public Response createImage() {
+        return Response.status(201).entity(Optional.empty()).build();
```

Comment:
Please check this status code for such cases
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501